cmd/testserver/main.go

```diff
@@ -0,0 +1,131 @@
+// Copyright 2023 Cloudflare, Inc.
```

Comment:
Reviewer note: Comparable to server/server.go in **this** repo. There is not comparable tool in the internal repo.